Gemfile

```diff
@@ -4,3 +4,5 @@ gem 'minitest'
 gem 'rubocop', '0.36.0'
 gem 'simplecov'
 gem 'rake'
+gem 'values'
```

Comment:
Yeah I don't suspect these 2 would cause any major issues - but I just know from experience dependency creep can be a big problem!